<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="CPDK196" failures="0" tests="2" name="Logintc.AlwaysRun" time="2.038" errors="1" timestamp="2024-02-23T13:01:29 IST" skipped="0">
  <testcase classname="Logintc.AlwaysRun" name="re1" time="2.035">
    <error message="Method Logintc.AlwaysRun.re1() didn&#039;t finish within the time-out 2000" type="org.testng.internal.thread.ThreadTimeoutException">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method Logintc.AlwaysRun.re1() didn't finish within the time-out 2000
at java.base@17.0.6/java.lang.Thread.sleep(Native Method)
at app//Logintc.AlwaysRun.re1(AlwaysRun.java:8)
at java.base@17.0.6/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base@17.0.6/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base@17.0.6/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base@17.0.6/java.lang.reflect.Method.invoke(Method.java:568)
at app//org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at app//org.testng.internal.invokers.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:47)
at app//org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:76)
at app//org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:11)
at java.base@17.0.6/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base@17.0.6/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base@17.0.6/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base@17.0.6/java.lang.Thread.run(Thread.java:833)
]]>
    </error>
  </testcase> <!-- re1 -->
  <testcase classname="Logintc.AlwaysRun" name="re2" time="0.003"/>
</testsuite> <!-- Logintc.AlwaysRun -->
